Gems/Atom/RHI/Code/Tests/FrameGraphTests.cpp

```diff
@@ -10,6 +10,8 @@
 #include <Tests/FrameGraph.h>
 #include <Tests/Factory.h>
 #include <Tests/Device.h>
+#include <Atom/RHI/MultiDeviceImagePool.h>
```

Comment:
Any reason why this is now multi-device?

Reply:
attachments are MD now